qiita_db/meta_util.py

```diff
@@ -184,12 +184,16 @@ def update_redis_stats():
     number_of_samples = {}
```

Comment:
Adding here cause this is the highest (smallest line number) github will allow me to add a message; studies is missing the status: 'awaiting_approval'; all the statuses are in 'qiita.visibility'. We could ignore, as we have been doing until now, add via qiita.visibility or just add to the dict in line 179.

Reply:
Shouldn't an artifact retain its prior state until approved?

Reply:
Well, the artifact state is awaiting_approval and I _think_ to accomplish what you suggest we will need another table where we have all sandbox artifacts that are awaiting for approval. Not sure if worth the dev time to accomplish this. As a reminder, when an artifact is created its status is sandbox, then awaiting_approval, then private and finally public.

Reply:
Does awaiting apptoval only ever mean sandbox -> private?

Reply:
Yup, an artifact to become private needs to be approved ...